### DISCUSSION TITLE
Print Speed On TFT display

### DISCUSSION THREAD

**Discussion by Kidabah** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2245] [Q&A]
Hi was wondering if anyone has been able to get or is there a possible way that you can get the print speed to display on TFT screen. Currently all i am able to see is the percentage of , not actual print speed. If i increase the print speed the display reflects i.i if i go from 100% to 200%. Would love to be able to see actual print mm per sec.
Regards
Chris

**Comment by GrahamJB** [comment]
That's not going to work.
There are many speeds used in 3D printing for layers, initial layers, walls...
The override changes them all by a percentage but to report which one is currently in effect and how it has been affected would be extremely complex and probably very confusing.
